### PR TITLE
feat(atsu): auto print when printer selected

### LINKED ISSUE #3198: [BUG] Printer should print with printer ouput selected
**Mod Version**
26th Jan 21 dev version

**Describe the bug**

When requesting weather no matter what type and output format is set to Printer, the message should print itself without going into the received messages. Only when its selected to MCDU it will not print.
image

**To Reproduce**

1.
2.
3.

**Expected behavior**

Message should be printed automatically

**Actual behavior**

Message does not print

**References**

IRL experience

**Additional context**

Was this working before/when did the issue start occurring?

Is this a problem in the vanilla unmodded game?

Discord username (if different from GitHub):

### PR DESCRIPTION
**Pull request by MisterChocker**
Fixes #3198 
Replaces #3199

## Summary of Changes

* Requesting ATIS with the "FORMAT FOR PRINTER" option selection will now cause the message to go straight to the printer,
company messages on ecam automatically disappears after a few seconds
* Changed default ATIS format from "PRINTER" to "MCDU"

## References

Based on feedback from @St54Kevin, a real world A320 pilot.

## Additional context

Discord username (if different from GitHub): DerL30N#3751

## Testing instructions

Request ATIS with the "FORMAT FOR" option set to "PRINTER" and observe:
- `company message` appears on ecam and automatically disappears
- message gets automatically printed,
- message appears in message list as `viewed`,

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

File: flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/ATSU/A320_Neo_CDU_AOC_RequestsAtis.js [hunk 1]
```diff
@@ -1,21 +1,28 @@
+/*
+ * A32NX
+ * Copyright (C) 2020-2021 FlyByWire Simulations and its contributors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
```
Comment (tracernz) [inline flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/ATSU/A320_Neo_CDU_AOC_RequestsAtis.js:7]: Have we switched back to GPL? (I hope so! 😄 )

Comment (MisterChocker) [reply flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/ATSU/A320_Neo_CDU_AOC_RequestsAtis.js:7]: ehm xD I in my head did LOL

probably a "left over" from tylers pr his branch was quite old..

**Review by wpine215 — APPROVED** [review]

**Comment by St54Kevin** [comment]
**Quality Assurance Tester**

Object of testing: #5146 
Tier of Testing: 1
Date: 19/06/2021

**Testing Process:**
1. Requested ATIS of an online airport with MCDU selected
2. Request with PRINTER selected

**Negatives:** N/A
**Testing Results:** Passed

Thanks for adressing this so fast :) Love the dedication <3

**Comment by frankkopp** [comment]
Quality Assurance Tester Report

Discord: Cdr_Maverick#6475
Object of testing: #5146
Tier of Testing: 1
Date: 20.6.2021

Testing Process: 
Requested ATIS from several airports while at gate.
Test MCDU and PRINTER format

Negatives: None
Testing Results: Passed
Conclusions: LGTM

I love these small quality of live features :)

**Comment by AviationSpotter** [comment]
Quality Assurance Trainee Report

Discord: Aviation Spotter#9102
Object of testing: #5146
Tier of Testing: 1
Date: 20/06/2021

Testing Process:

1. Requested ATIS, used printer option, and four different airports.
2. Monitored ECAM message "COMPANY MSG" and printer

Negatives: N/A
Testing Results: Passed

Conclusions: Didn't run into any issues! :)

**Comment by dragold** [comment]
Category: QA Tester trainee
Name: Mr.Blank#2638
Date of testing:06/21/2021
The version of the sim: 1.17.3.0
PR Tested: #5146
Tier of testing: 1

Changes to observe:
-Requesting ATIS with the "FORMAT FOR PRINTER" option selection will now cause the message to go straight to the printer,
company messages on ecam automatically disappears after a few seconds.
-Changed default ATIS format from "PRINTER" to "MCDU".

Testing instructions
Request ATIS with the "FORMAT FOR" option set to "PRINTER" and observe:

-company message appears on ecam and automatically disappears
-message gets automatically printed,
-message appears in message list as viewed

Testing technique: Tryout

Methodology:
- (SOP)
-- ATIS Request,(at gate)
-- CDU and PRINTER format Test

Negatives: NIL
Testing Results: Passed

Conclusions: All observations whereas expected

**Pull request merged**